### ISSUE TITLE
typescript GlobalQuasarLanguage and GlobalQuasarIconSet not exported

### ISSUE THREAD

**Issue by fuzzybook** [quasarframework/quasar#7407]
node_modules\quasar\dist\types\index.d.ts

line 17 & 23
```
export interface GlobalQuasarLanguage extends QuasarLanguage {
  set(lang: QuasarLanguage): void;
  /** Returns undefined when in SSR mode or when it cannot determine current language. */
  getLocale(): string | undefined;
}

export interface GlobalQuasarIconSet extends QuasarIconSet {
  set(iconSet: QuasarIconSet): void;
}
```

node_modules\quasar\dist\types\globals.d.ts
line 7944
`import { QVueGlobals, GlobalQuasarLanguage, GlobalQuasarIconSet } from "./globals";
`

**Comment by bekliev** [comment]
Same here:
![image](https://user-images.githubusercontent.com/4066435/87201828-12283100-c308-11ea-84da-f1dc0c32c32a.png)

Getting it right after the cli installation has been completed

**Comment by bekliev** [comment]
Same here...

Getting it right after the cli installation has been completed

![image](https://user-images.githubusercontent.com/4066435/87201921-3ab02b00-c308-11ea-97a0-67d63f719691.png)

### Versions
* @quasar/cli@1.1.0
* quasar@1.12.9

**Comment by bekliev** [comment]
Oh, I see that you've already fixed that in PR https://github.com/quasarframework/quasar/pull/7403
Exactly in this commit: https://github.com/quasarframework/quasar/commit/81dec232bbc77e19b9ad01ef647a7143d1303dfa

Waiting for new release :smile: 👍

**Comment by webnoob** [comment]
I'll be sorting a patch for this shortly.

**Comment by webnoob** [comment]
Fixed in latest: quasar-v1.12.10
